vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -41,8 +41,8 @@ const (
 	devWaitTimeout   = 1 * time.Second
 	sleepBeforeMount = 1 * time.Second
 	watchPath        = "/dev/disk/by-path"
-	version          = "VMDK Volume Driver v0.3"
-	driverName       = "vmdk"
+	version          = "vSphere Volume Driver v0.3"
```

Comment:
Notes in docs/user-guide/docker-plugin-drivers.md is updated to mention the change and deprecation of the vmdk driver name. 

For version number, which number should we use?